### ISSUE TITLE
github and wowace

### ISSUE THREAD

**Issue by flesk** [gdraynz/wowui#7]
i see you already have git module and im gonna try adding a conf manually?

also if you do curseforge and have no problem with cloudflare adding wowace too would be really a feature.

thank you!

**Comment by gdraynz** [comment]
Hey!
I hid the github module because it's kind of weird since right now it depends on the repo file structure but I guess I could add it back.
No idea about the cloudflare thing, I simply tried to use the APIs behind curseforge and wowinterface that are already used behind the scene. I didn't see anything I could really use for WoWAce though...

**Comment by flesk** [comment]
yeah but most github repos should work out of the box, just need to be able to select the branch, for wow its pretty easy because you can align/guess the folder structures by the .toc files. maybe i can get a quick jscript env running to test.

also pretty lucky to have not been bothered by cloudflare yet. many other clients hit 503/404s after hitting the limits

we dont know yet but it seems to be, the pattern and the frequency of the requests. so consider this with the "UPDATE ALL" feature!
